content/tabular-data-package/contents.lr

```diff
@@ -65,6 +65,7 @@ Tabular Data Package builds directly on the [Data Package][dp] specification. Th
 Tabular Data Package has the following requirements over and above those imposed by [Data Package][dp]:
 
 - There `MUST` be at least one `resource` in the `resources` `array`
+- There `MUST` be a `profile` property with the value `tabular-data-package`
```

Comment:
Do we need to link to profiles spec here in some way so people know where this is coming from?